browser/pages/confirm/controller.js

```diff
@@ -20,6 +20,11 @@ class ConfirmController {
     $scope.checkboxModel = {};
     $scope.platform = Platform.OS;
     $scope.detectionStyle = false;
+    $scope.virtualization = true;
+    Platform.isVirtualizationEnabled().then(status => {
```

Comment:
It looks good, but should be running along with other detectors at line 74 down below.